exercises/sum.cpp

```diff
@@ -11,22 +11,21 @@ using namespace std;
 
 int sum(int a, int b) {return a + b;}
 
-bool toRepeat(char ch){
-  if(ch=='y') return true;
-  return false;
-}
-
-int main(){
-  char choise='n';
+void makeSum(){
+  char choice='n';
   int n,m;
   do{
     cout << "Insert the first number: ";
     cin >> n;
     cout << "inserisci il secondo addendo: ";
     cin >> m;
```

Comment:
troppe lingue in una volta

comunque, potresti spostare anche queste due in una funzione separata, come qui https://github.com/UNICT-Quality-Development/git-course-exercises/pull/12 e, leggi bene i commenti di quella PR per ottimizzare il codice

Reply:
Sono andato a vedere le altre PR, spero di aver colto quanti più consigli possibil, grazie!